### DISCUSSION TITLE
Get and set content inside node view (v2)

### DISCUSSION THREAD

**Discussion by volarname** [ueberdosis/tiptap#1704] [General Questions]
Hello
I need to access content inside NodeView (<node-view-wrapper>). Of course I need to get and set the data. Not by inline editing using <node-view-content>, but programatically. How?
Thank you

**Comment by jloganolson** [comment]
Could you provide some context or an example of how you've tried to set/get without succeeding? Are you using collaboration (i.e. yxmlfragments) or "vanilla" tiptap?
